platform/web/audio_driver_web.cpp

```diff
@@ -186,6 +187,192 @@ Error AudioDriverWeb::input_stop() {
 	return OK;
 }
 
+bool AudioDriverWeb::is_stream_registered_as_sample(const Ref<AudioStream> &p_stream) const {
+	ERR_FAIL_COND_V_MSG(p_stream.is_null(), false, "Parameter p_stream is null.");
+	return godot_audio_sample_stream_is_registered(itos(p_stream->get_instance_id()).utf8().get_data()) != 0;
+}
+
+void AudioDriverWeb::register_sample(const Ref<AudioSample> &p_sample) {
+	ERR_FAIL_COND_MSG(p_sample.is_null(), "Parameter p_sample is null.");
+	ERR_FAIL_COND_MSG(p_sample->stream.is_null(), "Parameter p_sample->stream is null.");
+
+	String loop_mode;
+	switch (p_sample->loop_mode) {
+		case AudioSample::LoopMode::LOOP_DISABLED: {
+			loop_mode = "disabled";
+		} break;
+
+		case AudioSample::LoopMode::LOOP_FORWARD: {
+			loop_mode = "forward";
+		} break;
+
+		case AudioSample::LoopMode::LOOP_PINGPONG: {
+			loop_mode = "pingpong";
+		} break;
+
+		case AudioSample::LoopMode::LOOP_BACKWARD: {
+			loop_mode = "backward";
+		} break;
+	}
+
+	double length = p_sample->stream->get_length();
+	switch (p_sample->loop_mode) {
+		case AudioSample::LoopMode::LOOP_FORWARD:
+		case AudioSample::LoopMode::LOOP_PINGPONG:
+		case AudioSample::LoopMode::LOOP_BACKWARD: {
+			// TODO: Calculate length based on `loop_begin` / `loop_end`.
+		} break;
```

Comment:
I assume that TODO is still valid, if so I'd suggest opening an issue to keep track of it.

Reply:
Finally, I think it was leftover code. I removed it. Thanks!